frontend/src/users/pages/menu_page_/menuPage.js

```diff
@@ -0,0 +1,104 @@
+// import {useEffect,useState} from "react";
+import {useEffect} from "react";
+import WorkoutDetails from "../../components/WorkoutDetails";
+import Navbar3 from "../../components/Navbar3";
+// import ItemForm from "../components/Itemform"
+import ItemForm from "../../components/Itemform";
+
+// const MenuPage=()=>{
+
+//     const [menuitems,setMenuItems]=useState(null);
+//     useEffect(()=>{
+//         const fetchWorkouts=async()=>{
+//         const response=await fetch('/canteen/food',{
+//             method:"GET"
+//         })
+//         const json=await response.json();
+//         if(response.ok){
+//             setMenuItems(json);
+//         }
+//         }
+//         fetchWorkouts();
+//     },[])
+//     return(
+//         <>
+//         <Navbar3 />
+//         <div className="home">
+//             <div className="workouts">
+//                 {menuitems && menuitems.map((workout)=>(
+//                   <WorkoutDetails key={workout._id} workout={workout}/>
+//                 )
+
+//                 )}
+//             </div>
+//             {/* <ItemForm /> */}
+//         </div>
+//         </>
+//     )
+// }
+// export default MenuPage;
+
+
+
+
+import React, {useState} from "react";
+import './menuPage.css'
+
+
+// import Cart from "../components/Cart";
+import Cart from "../../components/Cart";
+// import FoodItems from "../components/FoodItems";
+import FoodItems from "../../components/FoodItems";
+// import Navbar3 from '../../components/Navbar3'
+
+
+export default function MenuPage(){
+    const [cart, setCart]=useState([]);
+    
+    const handleClick= (item)=>{
+        let isPresent=false;
+        cart.forEach((product)=>{
+            if(item.id===product.id){
+                isPresent =true;
+            }
+        })
+        if(isPresent){
+            return;
+        }
+        else {  
+            setCart([...cart,item]);
+        }
+    }
+
+    const handleChange =(item, d)=>{
+        let ind = -1;
+		cart.forEach((data, index)=>{
+        
```

Comment:
Use inbuilt array methods like cart.find() or cart.indexOf() (google them) instead of manually implementing the functions. If you just wanted practice, okay, but just saying, avoid it for future projects. 
Later on there will be a time when there's so many more complicated logic that you would have to implement, that you will simply just use the inbuilt methods instead of implementing them on your own ;-)